Makefile

```
@@ -74,6 +74,10 @@ lint: ## Run golangci-lint over the codebase.
 test: generate fmt vet ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path --bin-dir $(PROJECT_DIR)/bin)" ./hack/test.sh
```

To avoid drift between this and the test-only target, we can make test rely on the test-only target
```suggestion
test: generate fmt vet test-only ## Run tests.
```

---

yeah, makes sense